GoogleDataTransport/GDTCORLibrary/Public/GoogleDataTransport/GDTCORStorageProtocol.h

```diff
@@ -25,6 +25,8 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+typedef uint64_t GDTCORStorageSizeBytes;
```

Comment:
Why do clients need this?

Reply:
@paulb777 The type was introduced to help with the API readability. It stresses the value meaning and units. I think this should help with preventing issues like we had with timestamps #6102.